.npmrc

```diff
@@ -0,0 +1 @@
+registry=https://registry.npmjs.org/
```

Comment:
should this file be placed under federated gateway and website directories instead (same as .nvmrc)?

Reply:
Sure, I can move it